src/apps/mpx.erl

```diff
@@ -58,20 +62,20 @@ handle_client_message(_,_) -> ok.
 
 client_disconnected(Client) -> gen_server:call(?SERVER, {client_disconnected, Client}).
 
-handle_call({subscribe, #de_client{meta=Apps, data=AppData}=Client, App}, _, State) ->
+handle_call({subscribe, #de_client{meta=Apps, data=AppData}=Client, App, Data}, _, State) ->
```

Comment:
Line 65 is too long: handle_call({subscribe, #de_client{meta=Apps, data=AppData}=Client, App, Data}, _, State) ->.